README.md

```diff
@@ -4,31 +4,25 @@ Midtrans Drupal 8/9 Commerce Payment Gateway Module
 Midtrans :heart: Drupal 8/9!
 Let your Drupal Commerce 2 store integrated with Midtrans payment gateway.
 
-### Description
+#### Description
 This is the official Midtrans extension for the Drupal Commerce 2 E-commerce platform.
 
-### Version
+#### Version
 2.1.0
 (for Drupal v8.x and Drupal v9.x)
 
-### Requirements
-The following plugin is tested under following environment:
+#### Requirements
+The following module is tested under following environment:
 
 * PHP v5.6.x or greater
 * MySQL version 5.0 or greater
+* [Midtrans PHP Library](https://github.com/Midtrans/midtrans-php)
 * [Drupal v8.x or greater](https://www.drupal.org/project/drupal)
 * [Drupal Commerce 8.x-2.xx ](http://www.drupal.org/project/commerce)
 
-#### Composer Installation
-If you are using [Composer](https://getcomposer.org), you can install via composer CLI
-run: `composer require drupal/midtrans_commerce`  on your terminal.
-
-#### Manual Instalation
-The manual installation method involves downloading our feature-rich plugin and uploading it to your webserver via your favourite FTP application.
-
-1. Download the plugin file to your computer and unzip it, rename folder to ``commerce_midtrans``.
-2. Using an FTP program, or your hosting control panel, upload the unzipped plugin folder to your Drupal modules installation's ``[Drupal folder]/modules/contrib/`` directory.
-
+#### Installation
```

Comment:
Can this be installed from Drupal UI from module store? can you please update with how to?

Reply:
unfortunately, there is no UI in drupal admin to search module and install directly from the drupal store

Reply:
I see, what about this https://www.drupal.org/docs/user_guide/en/extend-module-install.html#s-using-the-administrative-interface will that work for our module or not? @omarxp

Reply:
never use it before, I just now installed it, but still have not found the form to install from the URL.

Reply:
ok then, that method of install is optional for now.